app/views/hub/forced_password_resets/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="slab">
+  <div class="grid__item width-one-half shift-one-fourth">
+    <div class="form-card form-card--compact">
+      <div class="form-card__header">
+        <h1 class="form-card__title">Please update your password. </h1>
+        <p class="text--help">It’s important that you keep this password secure and do not share it with anyone.
+          Please keep all client information safe and confidential.</p>
+      </div>
+      <%= form_for(@user, builder: VitaMinFormBuilder, url: { action: :update }, html: { method: :put }) do |f| %>
+        <div class="form-card__content">
+          <%= f.cfa_input_field(:password, "New password", type: "password", classes: ['form-width--long', 'password']) %>
+          <%= f.cfa_input_field(:password_confirmation, "Confirm new password", type: "password", classes: ['form-width--long', 'password']) %>
+          <div class="form-card__footer">
+            <%= f.continue "Update" %>
+          </div>
+        </div>
+      <% end %>
+    </div>
+  </div>
+</div>
```

Comment:
Totally unimportant: Asheesh likes it when files end in newline characters, but no one else on the team seems to care, so do with this whatever you will. There's a RubyMine/IntelliJ option for this called, "Ensure every saved file ends with a line break".